ui/lib/kv/addon/components/kv-data-fields.js

```diff
@@ -23,7 +23,7 @@ import { stringify } from 'core/helpers/stringify';
  * @param {boolean} showJson - boolean passed from parent to hide/show json editor
  * @param {object} [modelValidations] - object of errors.  If attr.name is in object and has error message display in AlertInline.
  * @param {callback} [pathValidations] - callback function fired for the path input on key up
- * @param {boolean} [isEdit=false] - if true, this is a new secret version rather than a new secret. Used to change text for some form labels
+ * @param {boolean} [type=null] - can be edit, create, or details. Used to change text for some form labels
```

Comment:
this should have been change to 'string'. also FWIW null default values should not be added to JSDOC
```suggestion
 * @param {string} [type] - can be edit, create, or details. Used to change text for some form labels
```